kolibri/plugins/style_guide/assets/src/views/shell/component_api.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+
+  <div>
+
+    <h3>Require Path</h3>
+    <p>{{ requirePath }}</p>
```

Comment:
nice!

Might possibly even make it more explicit as a copy-able code block. E.g.:

```html
<h3>Require statement</h3>
<p><code>require('{{ requirePath }}')</code></p>
```